### PR TITLE
[DO-1258] Quality alert details page

### PR DESCRIPTION
**Pull request by viannbreslerAccso**
Implementation of the quality alert figma designs

### CODE REVIEW

**Review by JoelDietz — CHANGES_REQUESTED** [review]
Please check the status of the quality alerts overlays again (or are they part of a different ticket, that was not part of this sprint?)
Also there is a problem with the back button (leaving the detail page) see comment. And last the cancel button we wondered in the review about should be only visible when the approve button is also visible. (see the corresponding comment.)
Everything else looks good.

File: frontend/src/assets/locales/en/common.json
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/assets/locales/en/common.json:1]: Done

File: frontend/src/assets/locales/en/common.json [hunk 1]
```diff
@@ -248,15 +249,15 @@
     },
     "modal": {
       "acceptTitle": "Accept of alert",
-      "acceptReasonHint": "Enter the reason for accepting this alert.",
+      "acceptReasonHint": "Enter the reason for accepting this alert",
       "acknowledgeTitle": "Acknowledgment of alert",
       "approvalTitle": "Approval of alert",
```
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/assets/locales/en/common.json:254]: Done

File: frontend/src/assets/locales/en/common.json [hunk 2]
```diff
@@ -248,15 +249,15 @@
...
       "closeTitle": "Close of alert",
```
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/assets/locales/en/common.json:255]: Done

File: frontend/src/app/modules/page/alerts/detail/alert-detail.component.html [hunk 3]
```diff
@@ -16,197 +16,218 @@
...
+          supplierPartsDetailInformation$ | async;
+          main: mainSupplierPartsTmp;
+          error: errorTmp;
+          loading: loaderTmp
+        "></ng-container>
+      </div>
+    </div>
+  </div>
+
+  <div class="buttons-container">
```
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/app/modules/page/alerts/detail/alert-detail.component.html:112]: Done - Yes, I forgot to uncomment the ngIf

File: frontend/src/app/modules/shared/components/notification-reason/notification-reason.component.scss [hunk 4]
```diff
@@ -40,19 +44,19 @@
...
   position: relative;
   padding: 10px 20px;
-  border-radius: 25px;
+  border-radius: 10px;
 }
 
 .message--text__date {
   font-weight: 600;
+
   &-icon {
```
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/app/modules/shared/components/notification-reason/notification-reason.component.scss:60]: Done - That css is meant for the icon. But I updated the 10px to 12px of the other section which fixed the size of the text next to the clock icon

File: frontend/src/app/modules/page/alerts/detail/alert-detail.component.html [hunk 5]
```diff
@@ -16,197 +16,218 @@
...
+      <h3 class="mb-4 pl-2 flex items-center">
+        <img class="pr-2"
+          src="/assets/images/icons/chevron_left.svg"
+          alt="<" />
+        <span class="header-description">
+          {{ selectedAlert?.description }}
+        </span>
+      </h3>
+    </div>
+    <img class="close-button"
```
[the remark this answers is not in the document]
Comment (JoelDietz) [reply frontend/src/app/modules/page/alerts/detail/alert-detail.component.html]: Is a browser thing. In chrome it works and in firefox it does not

**Review by JoelDietz — APPROVED** [review]

**Pull request merged**